scripts/script.js

```diff
@@ -53,6 +54,12 @@ cocktailApp.prepareSubmitListener = function () {
 };
 
 cocktailApp.getRandomCocktail = function() {
+  // adding pop-up function
+  cocktailApp.popup.classList.add('popup__container--open')
+  const popupButton = this.popup.querySelector('.popup__button');
```

Comment:
This element is always around, but hidden, right? Can we add **popupButton** to the cocktailApp namespace?